app/league/admin/views.py

```diff
@@ -1,10 +1,12 @@
 # -*- coding: utf-8 -*-
 """User views."""
-from flask import Blueprint, flash, redirect, render_template, request, url_for
+from flask import Blueprint, flash, redirect, render_template, request, \
```

Comment:
Alphabetical order, and use `()` instead of `\`

```
from flask import (Blueprint, current_app, flash, redirect, render_template,
                               request, url_for)
```